app/lib/advanced_filter_constants.rb

```diff
@@ -0,0 +1,472 @@
+            'monitorees with “Close Contact with a Known Case”. Use commas to separate multiple values (ex: “12, 45” will return records where known Case ID '\
+            'is “45” or “45, 12”). Leaving this field blank will return monitorees with missing and null values.',
+          Contains:
+            'Returns records that contain a user-entered search value when the known Case ID is specified for monitorees with “Close Contact with '\
+            'a Known Case”. Use commas to separate multiple values (ex: “12, 45” will return records where known Case ID is “123, 90” or “12” or “1451). '\
+            'Leaving this field blank will return monitorees with missing and null values.'
+        }
+      },
+      {
+        name: 'email',
+        title: 'Primary Contact Email (Text)',
+        description: 'Monitoree primary contact email address',
+        type: 'search'
+      },
+      {
+        name: 'sara-id',
+        title: 'Sara Alert ID (Text)',
+        description: 'Monitoree Sara Alert ID',
+        type: 'search'
+      },
+      {
+        name: 'telephone',
+        title: 'Primary Contact Telephone Number (Text)',
+        description: 'Monitorees with a primary contact telephone number',
+        type: 'search',
+        options: ['Exact Match', 'Contains']
+      },
+
+      # SELECT FILTER OPTIONS
+      {
+        name: 'monitoring-plan',
+        title: 'Monitoring Plan (Select)',
+        description: 'Monitoree monitoring plan',
+        type: 'select',
+        options: [
+          'None',
+          'Daily active monitoring',
+          'Self-monitoring with public health supervision',
+          'Self-monitoring with delegated supervision',
+          'Self-observation',
+          ''
+        ]
+      },
+      {
+        name: 'preferred-contact-method',
+        title: 'Primary Contact Preferred Reporting Method (Select)',
+        description: 'Monitoree primary contact preferred reporting method',
+        type: 'select',
+        options: ['Unknown', 'E-mailed Web Link', 'SMS Texted Weblink', 'Telephone call', 'SMS Text-message', 'Opt-out', '']
+      },
+      {
+        name: 'preferred-contact-time',
+        title: 'Primary Contact Preferred Contact Time (Select)',
+        description: 'Monitoree primary contact preferred contact time',
+        type: 'select',
+        options: ['Early Morning', 'Morning', 'Afternoon', 'Evening', 'Late Night', ''],
+        tooltip: ['Early Morning (Midnight - 7:00)', 'Morning (8:00 - 11:00)', 'Afternoon (Noon - 15:00)', 'Evening (16:00 - 19:00)',
+                  'Late Night (20:00 - 23:00)']
+      },
+      {
+        name: 'primary-language',
+        title: 'Primary Language (Select)',
+        description: 'Monitoree primary language',
+        type: 'select',
+        options: Languages.all_languages.values.sort_by do |lang|
```

Comment:
since the supported languages isn't something that varies between users, can we set this as `SUPPORTED_LANGUAGES` or some other constant outside of this method?